source/entrance.cpp

```diff
@@ -436,6 +442,165 @@ static void ShuffleEntrancePool(std::vector<Entrance*>& entrancePool, std::vecto
   }
 }
 
+std::unordered_map<s16, std::string> entranceNames = {
+  { 0x0000, "Kokiri Forest Deku Tree Entrance" },
+  { 0x0209, "Deku Tree" },
+  { 0x0004, "Death Mountain Trail Dodongo's Cavern Entrance" },
+  { 0x0242, "Dodongo's Cavern" },
+  { 0x0028, "Zora's Fountain Jabu Jabu's Belly Entrance" },
+  { 0x0221, "Jabu Jabu's Belly" },
+  { 0x0169, "Sacred Forest Meadow Forest Temple Entrance" },
+  { 0x0215, "Forest Temple" },
+  { 0x0165, "Death Mountain Crater Fire Temple Entrance" },
+  { 0x024A, "Fire Temple" },
+  { 0x0010, "Lake Hylia Water Temple Entrance" },
+  { 0x021D, "Water Temple" },
+  { 0x0082, "Desert Colossus Spirit Temple Entrance" },
+  { 0x01E1, "Spirit Temple" },
+  { 0x0037, "Graveyard Warp Pad" },
+  { 0x0205, "Shadow Temple" },
+  { 0x0098, "Kakariko Village Bottom of the Well Entrance" },
+  { 0x02A6, "Bottom of the Well" },
+  { 0x0088, "Zora's Fountain Ice Cavern Entrance" },
+  { 0x03D4, "Ice Cavern" },
+  { 0x0008, "Gerudo Fortress Training Grounds Entrance" },
+  { 0x03A8, "Gerudo Training Grounds" },
+  { 0x0433, "Kokiri Forest at Mido's House" },
+  { 0x0443, "Mido's House" },
+  { 0x0437, "Kokiri Forest at Saria's House" },
+  { 0x0447, "Saria's House" },
+  { 0x009C, "Kokiri Forest at House of Twins" },
+  { 0x033C, "House of Twins" },
+  { 0x00C9, "Kokiri Forest at Know-It-All House" },
+  { 0x026A, "Know-It-All House" },
```

Comment:
I think it'd be better if instead of `at` it said `to` or `from`, like you did further below.